stories/src/repositories/story.repository.ts

```diff
@@ -1,31 +1,33 @@
-import { v4 } from "uuid";
+import { v4, validate as isUuid } from "uuid";
 import { IDatabase } from "../types/database.type";
-import { ICreateStory } from "../types/story/dtos/createStory.dto";
-import { IUpdateStory } from "../types/story/dtos/updateStory.dto";
-import { IStory } from "../types/story/entities/story.entity";
+import { CreateStoryDTO } from "../dtos/createStory.dto";
+import { UpdateStoryDTO } from "../dtos/updateStory.dto";
+import { StoryEntity } from "../entities/story.entity";
 
 export class StoryRepository {
     constructor(private readonly database: IDatabase) {}
-    async listAll(): Promise<IStory[]> {
+    public async listAll(): Promise<StoryEntity[]> {
         return (await this.database.tables.Story.findAll()).map((story) => story.dataValues);
     }
-    async findById(uuid: string): Promise<IStory> {
+    public async findById(uuid: string): Promise<StoryEntity> {
         return (await this.database.tables.Story.findOne({ where: { id: uuid } }))?.dataValues;
     }
-    async create(story: ICreateStory): Promise<IStory> {
-        return (await this.database.tables.Story.create({ ...story, id: story.id || v4() }))
-            ?.dataValues;
+    public async create(story: CreateStoryDTO): Promise<StoryEntity> {
+        return (await this.database.tables.Story.create({ ...story, id: v4() }))?.dataValues;
     }
-    async update(uuid: string, updateInfo: IUpdateStory): Promise<IStory> {
+    public async update(uuid: string, updateInfo: UpdateStoryDTO): Promise<StoryEntity> {
         await this.database.tables.Story.update(updateInfo, { where: { id: uuid } });
```

Comment:
Aqui há várias referências para `this.database.tables.Story.OPERATION`, acho que vale a pena criar uma referência apenas para `Story` diretamente. Assim seu código faria apenas: `this.stories.OPERATION`